float32[8] joint_pos
float32[8] joint_vel
float32[8] joint_cur
int32 mode